test-data/unit/check-modules.test

```diff
@@ -2068,7 +2068,6 @@ def __getattr__(name): ...
 [builtins fixtures/module.pyi]
 
 [case testModuleLevelGetattrNotStub36]
```

Comment:
This is testing that module-level `__getattr__` isn't recognised by mypy with --python-version set to 3.6 or lower. If we no longer support 3.6 or lower, we should just delete the test